src/main/java/io/anserini/ltr/feature/ConfidencePooler.java

```diff
@@ -0,0 +1,30 @@
+package io.anserini.ltr.feature;
+
+import java.util.List;
+
+public class ConfidencePooler implements Pooler  {
```

Comment:
"Pooler" or "Pooling"?

Reply:
I suppose I understand what you mean... a "pooler" does "pooling"?

Don't care either way, just a note.